tensorflow/python/keras/backend_test.py

```diff
@@ -1736,5 +1736,34 @@ def test_batch_normalization(self):
     self.assertEqual(normed.shape.as_list(), [10, 3, 5, 5])
 
 
+class ControlOpsTests(test.TestCase):
+
+  def test_function_switch_basics(self):
+    pred = keras.backend.placeholder([], dtype='bool')
+
+    def false_fun():
+      return array_ops.constant(False)
+
+    with self.cached_session():
+      x2 = keras.backend.switch(pred, lambda: array_ops.constant(True), false_fun)
```

Comment:
Just curious why the true branch is a lambda, while the false branch is a function, can we use lambda for both case for simplicity?

Reply:
Sure, same is updated in the code.

Regards
Amit